src/store/modules/recent/getters.ts

```diff
@@ -0,0 +1,79 @@
+import { Getter } from 'vuex';
+import { find } from 'lodash';
+import { MENU_ICON } from '@/common/modules/navigations/gnb/config';
+import {
+    RECENT_TYPE, RecentItem, RecentState,
+} from '@/store/modules/recent/type';
+import { getAllMenuList, getParents } from '@/lib/helper/menu-helper';
+
+
+export const menuItems: Getter<RecentState, any> = (state: RecentState, getters, rootState): RecentItem[] => {
+    const menuList = getAllMenuList(rootState.display.menuList);
+    const results: RecentItem[] = [];
+    state.menuItems.forEach((d) => {
+        const menu = find(menuList, { id: d.itemId });
+        if (menu) {
+            results.push({
+                itemType: RECENT_TYPE.MENU,
+                itemId: menu.id,
+                name: menu.id,
+                label: menu.label,
+                icon: MENU_ICON[menu.id.split('.')[0]],
```

Comment:
결국 menu.parents의 0번 인덱스의 id 이거나, menu.parents가 없으면 현재 아이템의 id를 구하고 싶은 것 같네요.
개인적으로는 menu의 id 구조를 알지 못하면, 이 부분은 이해하기 어려울 것 같아요.

아래처럼 바꾸면 어떨까요.
```suggestion
                icon: MENU_ICON[menu.parents[0]?.id ?? menu.id],
```

Reply:
오 좋습니다!!! parents가 이제 array 타입으로 들어가니 가능할듯요!
이렇게 바꾸겠습니다~